nmostesting/suites/BCP0060102Test.py

```diff
@@ -840,13 +846,14 @@ def test_03(self, test):
                     question, possible_answers, test_type='multi_choice', multipart_test=i)['answer_response']
 
                 if len(actual_answers) != len(expected_answers):
-                    return test.FAIL('Incorrect Receiver identified for Compatibility Set ' + sender['capability_set']
+                    return test.FAIL('Unable to identify compatible Receivers for Compatibility Set '
+                                     + sender['capability_set']
```

Comment:
Two things here:

* I want to understand this better. Sender's don't (yet... IS-11) have Constraint Sets (or Compatibility Sets... what is a compatibility set?)
* I think "Unable to identify" is still a little misleading, if there may be multiple compatible Receivers in the available choices, since it might be that they were unable to identify ANY compatible Receivers or just that they didn't identify all of them? Maybe just need to add "all", or to phrase it similar to above, like "Compatible Receivers not identified for"?

Any changes would then ripple through in other similar messages.

Reply:
The Capability Set is the TR-08 Capability Set if that helps.

Yes I like your suggestion for improving the wording.

Reply:
Ohhh, of course! Thanks for explanation!!! If we're going to use it in the messages to the user, we probably ought to spell that out, if you don't do so already :-)

Reply:
OK, now that I've read the code better, sorry again, how about something like:

```suggestion
                    return test.FAIL('Compatible Receivers not identified for '
                                     + 'Sender ' + sender['display_answer'] + ': '
                                     + 'Capability Set ' + sender['capability_set']
```